intro-to-rails-2-relationships/nested-routes.md

```diff
@@ -89,12 +177,50 @@ class BooksController < ApplicationController
 end
 ```
 
-<details>
-  <summary>
-    <strong>Question:</strong> What should our code do if the author is not found, that is, if the user goes to <code style="white-space: nowrap;">/authors/789012/books</code> or <code>/authors/toaster/books</code>?
-  </summary>
-  Though the above code does not handle this case, if we were doing this for a project, we would want to make sure that we respond with a not_found (404) response code in this case.
-</details>
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
+
+### !challenge
+
+* type: multiple-choice
+* id: 09f3c6d2-78c2-4e91-b1de-ca422a7538a0
+* title: What about if the author isn't found?
+* points: 1
+* topics: rails, rails-routes
+
+##### !question
+
+<strong>Question:</strong> What should our code do if the author is not found, that is, if the user goes to <code style="white-space: nowrap;">/authors/789012/books</code> or <code>/authors/toaster/books</code>?
```

Comment:
Oh, I've been suggesting deleting these "Question" strings but it looks like you intentionally added it here...It's obviously not a big deal but just seems like an odd thing to have in some lessons and not others and seems redundant.

Reply:
I had left them in because learn makes the question text gray, which annoys me to no end on accessibility concerns.